api/v1/tetragon/sensors.proto

```diff
@@ -38,6 +38,26 @@ message ListSensorsResponse {
 	repeated SensorStatus sensors = 1;
 }
 
+message ListTracingPoliciesRequest { }
+
+message TracingPolicyStatus {
+	// id is the id of the policy
+	uint64 id = 1;
+	// name is the name of the policy
+	string name = 2;
+	// namespace is the namespace of the policy (or empty of the policy is global)
+	string namespace = 3;
+	// info is additional information about the policy
+	string info = 4;
+	// sensors loaded in the scope of this policy
+	repeated string sensors = 5;
+
+}
+
+message ListTracingPoliciesResponse {
+	repeated TracingPolicyStatus policies = 1;
+}
```

Comment:
I know we don't support this now but do you see a case where a tracing policy is paused (loaded but inactive)? and if so I guess this can easily be added in future without breaking API?

Reply:
I don't see why we would need to break the API. We can just add a status field.